### PR TITLE
Adding endpoints for department>groups>users CRUD

### PR DESCRIPTION
**Pull request by rammano3**

### CODE REVIEW

File: help-fem.js [hunk 1]
```diff
@@ -629,6 +629,96 @@
...
+      getGroupUsers: function(data, callback) {
+        this.parent._doHyperJSONEmit(
+          this.parent._emitSessionEvent,
+          {type: 'organizationManager'},
+          {uri: '/groupusers', method: 'GET'},
+          data,
+          callback
+        );
+      },
+      createGroupUserTie: function(data, callback) {
```
Comment (calebthebrewer) [inline help-fem.js]: Just curious - whats `Tie` for?

Comment (rammano3) [inline help-fem.js]: ha, its something I guess I picked up from hydra organization manager... means Relation... I dont mind changing it... thoughts? I could just remove the word Tie... createGroupUser seems descriptive enough

Comment (calebthebrewer) [inline help-fem.js]: Either way it looks fine to me, I was just unfamiliar with that convention and noticed it didn't mirror the uri.

**Comment by calebthebrewer** [comment]
LGTM

**Comment by rammano3** [comment]
Added version bump

**Pull request merged**